src/components/views/voip/CallView.tsx

```diff
@@ -479,9 +485,12 @@ export default class CallView extends React.Component<IProps, IState> {
         let vidMuteButton;
         if (this.props.call.type === CallType.Video) {
             vidMuteButton = (
-                <AccessibleButton
+                <AccessibleTooltipButton
                     className={vidClasses}
                     onClick={this.onVidMuteClick}
+                    title={this.state.vidMuted ? _t("Start the camera") : _t("Stop the camera")}
+                    alignment={Alignment.Top}
+                    yOffset={-24}
```

Comment:
This number seems to be always the same, could it perhaps be a const at the top of the file?

Reply:
yeah, good point